content/en/docs/setup/independent/high-availability.md

```diff
@@ -1,536 +1,505 @@
 ---
 reviewers:
-- mikedanese
 - luxas
-- errordeveloper
-- jbeda
+- timothysc
+- detiber
+- chuckha
 title: Creating HA clusters with kubeadm
 content_template: templates/task
 ---
 
+{{% capture steps %}}'
```

Comment:
I need help here. I don't want this but if I don't include a non-empty steps section then I end up with a big error page

Reply:
edit nvm! taken care of